wordpress-mobile/WordPress-Android
# Issue/9349 update notifications tabs (Part 1)

### Fix
Update the ***Notifications*** list view from segmented control to tabs as described in https://github.com/wordpress-mobile/WordPress-Android/issues/9349.  This is the first part of the issue, which replaces the segmented control interface while keeping the same behavior.  The ability to swipe between tabs will be added in a subsequent pull request.  See the screenshots and animation below for illustration.

<img src="https://user-images.githubusercontent.com/3827611/53576644-68d90480-3b31-11e9-939e-0f4b299c7043.png">

<img src="https://user-images.githubusercontent.com/3827611/53672465-e55a0900-3c3f-11e9-9105-0910b0bb863c.gif" width="360">

### Test
1. Go to ***Notifications*** tab.
2. Notice ***All***, ***Unread***, ***Comments***, ***Follows***, and ***Likes*** tabs.
3. Tap ***All***, ***Unread***, ***Comments***, ***Follows***, and ***Likes*** tabs.
4. Notice notification list updates based on tab selected.
5. Scroll down/up.
6. Notice toolbar hides/shows based on scroll.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
The `RELEASE-NOTES.txt` document was updated in 96a8f2a with the following:
> Updated Notifications with tabs
